torch/export/_trace.py

```diff
@@ -707,13 +707,18 @@ def _maybe_fixup_gm_and_output_node_meta(old_gm, new_gm):
         + len(graph_signature.buffers)
         + len(graph_signature.input_tokens)
     )
+    fake_mode = detect_fake_mode(flat_fake_args)
     for node in gm.graph.nodes:
         if node.op == "placeholder":
             if index >= total_non_user_inputs:
                 user_arg = flat_fake_args[index - total_non_user_inputs]
                 if not isinstance(user_arg, torch.Tensor):
                     node.meta["val"] = user_arg
             index += 1
+        if node.op == "get_attr" and "val" not in node.meta:
+            val = getattr(gm, node.target)
```

Comment:
Would the node.target be in the format of "tensor_constant_0" or sth?

Reply:
Maybe assert that node.target is in the format where there is no delimeter in the name. For example, getattr(gm, foo.bar.const) will fail today.

Reply:
Same code is also here: https://github.com/pytorch/pytorch/blob/56da78f8faba7a7a9130caddd11c47bad23969b4/torch/_export/passes/lift_constants_pass.py#L159

Should both cases use a recursive getattr instead?

Reply:
Yeah just to be safe, i think we should. But probably doesn't matter in this case.